Makefile

```diff
@@ -0,0 +1,72 @@
+# Go parameters
```

Comment:
might be worth putting these in their own makefile also?

Reply:
don't think so at the moment. If the Makefile gets too big in the future we can probably take a look at it then